vip-jetpack/connection-pilot/class-jetpack-connection-controls.php

```diff
@@ -139,11 +139,24 @@ public static function connect_site( bool $skip_connection_tests = false, bool $
 	 */
 	public static function connect_akismet(): bool {
 		if ( class_exists( 'Akismet_Admin' ) ) {
+
 			if ( is_akismet_key_invalid() ) {
-				return \Akismet_Admin::connect_jetpack_user();
+				// Getting wpcomvip user, since it's the owner of the Jetpack connection
+				$current_user = wp_get_current_user();
```

Comment:
```suggestion
                                $current_user = wp_get_current_user();
				// Getting wpcomvip user, since it's the owner of the Jetpack connection
```

Reply:
Switching the lines to have comment just where you are getting the user:)